packages/components/notification/src/NotificationItem/NotificationItem.styles.ts

```diff
@@ -53,6 +53,7 @@ const titleStyle: CSSObject = {
 
 const contentStyle: CSSObject = {
   color: tokens.gray700,
+  wordBreak: 'break-word',
```

Comment:
Do we want a `hyphens: auto` in there as well?

Reply:
I don't see it changing anything 🤷🏼 
<img width="567" alt="Screenshot 2022-09-14 at 16 57 18" src="https://user-images.githubusercontent.com/6163988/190190104-bab35e05-fd64-46d8-9aaf-57078b47548d.png">

Reply:
It would be almost the same, but in some cases it would insert a hyphen which looks better and less broken:

> The browser is free to automatically break words at appropriate hyphenation points, following whatever rules it chooses.
> – https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens